src/averageAges.js

```diff
@@ -15,13 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
```

Comment:
The function name suggests it calculates the average age, but the function actually returns the average age of men. Consider renaming the function to better reflect its purpose.